Domain/Sia.Core/MVC/BaseController.cs

```diff
@@ -32,7 +32,7 @@ public abstract class BaseController : Controller
         public IActionResult OkIfFound<TResponse>(TResponse response)
         where TResponse : class
         {
-            if (response == null || response == "null")
+            if (response == null || response.ToString() == "null")
```

Comment:
I would rather take out the second clause here; I don't think we're ever going to get a hit on that case.

Reply:
👍